src/components/checkbox/checkbox.scss

```diff
@@ -7,12 +7,11 @@
     display: flex;
 
     .mdc-checkbox {
-        @include mdc-checkbox-ink-color(primary);
         @include mdc-checkbox-container-colors(
             secondary,
             on-primary,
-            secondary,
-            on-primary
+            primary,
+            primary
         );
```

Comment:
If we're going back to using MDC's standard styling, I would assume that we can simply remove these includes completely… unless you already tried that?

Reply:
I tried to remove them all, but the checkbox turned to a dark grey (when checked). Not sure why :/

Reply:
Hmm, interesting. But this one is needed, so it's 👌